polaris-react/src/components/DropZone/components/FileUpload/FileUpload.scss

```diff
@@ -27,7 +27,7 @@
   flex: 0 0 auto;
   border: none;
   border-radius: var(--p-border-radius-base);
-  padding: var(--p-space-1) 6px;
```

Comment:
Need approval from design on changing this one but seemed like a small enough change to get this to use our tokens and be on the grid (4px/8px) we want to use.

Reply:
Here are the Chromatic diffs (the bright green represents the space where the `padding` was removed):
<img width="1909" alt="Screen Shot 2022-04-18 at 2 21 48 PM" src="https://user-images.githubusercontent.com/21976492/163872223-20686bb0-664a-4af9-b141-6510d85224f8.png">
<img width="1184" alt="Screen Shot 2022-04-18 at 2 22 13 PM" src="https://user-images.githubusercontent.com/21976492/163872238-5839395a-41e2-49d1-9ea6-017087e5d0bd.png">